index.js

```diff
@@ -60,6 +62,9 @@ class DetailsMenuElement extends HTMLElement {
     for (const sub of subscriptions) {
       sub.unsubscribe()
     }
+
+    this.addEventListener('compositionstart', trackComposition)
+    this.addEventListener('compositionend', trackComposition)
```

Comment:
This should call `removeEventListener`. The subscription pattern in #42 will address this too.